tests/manage/monitoring/prometheus/test_rgw.py

```diff
@@ -25,10 +26,14 @@ def test_rgw_unavailable(measure_stop_rgw):
     # get alerts from time when manager deployment was scaled down
     alerts = measure_stop_rgw.get("prometheus_alerts")
     target_label = constants.ALERT_CLUSTEROBJECTSTORESTATE
-    target_msg = (
-        "Cluster Object Store is in unhealthy state for more than 15s. "
-        "Please check Ceph cluster health or RGW connection."
-    )
+    # The alert message is changed since OCS 4.7
+    if float(config.ENV_DATA["ocs_version"]) < 4.7:
```

Comment:
Could `ocs_version` here be also something like `4.7.2`? If yes, I don't think it will work.
Some code in ocs-ci uses https://pypi.org/project/semantic-version/ module (I just checked that it's installed as indirect dependency, so it will be available even though it's not directly listed in requirements of ocs-ci).

Reply:
Moreover consider what would happen when we reach version 4.10 :)

Reply:
right, please see https://github.com/red-hat-storage/ocs-ci/blob/76cf9dfa780f672fd95d78754f7fdcb19e633e66/ocs_ci/ocs/node.py#L991 as a reference